apps/app/src/test/scala/org/lfdecentralizedtrust/splice/integration/tests/BootstrapPackageConfigIntegrationTest.scala

```diff
@@ -42,6 +42,7 @@ import java.time.Instant
 import java.time.temporal.ChronoUnit
 import scala.jdk.CollectionConverters.*
 
+@org.lfdecentralizedtrust.splice.util.scalatesttags.NoDamlCompatibilityCheck
```

Comment:
I LOVE IT! 

Can we adda readme on how to add new tags and when to use them?  (can be done later in a different PR)

Reply:
did you see the readme I added?

Reply:
In DEVELOPMENT.md or something else that I'm missing?
I'm seeing there instructions to tag a test if it doesn't work with this setup but I don't see anything about how are tags used, when to add a new type of tag and how to make the test run aware of it.